command/job_deployments.go

```diff
@@ -71,13 +71,14 @@ func (c *JobDeploymentsCommand) AutocompleteArgs() complete.Predictor {
 func (c *JobDeploymentsCommand) Name() string { return "job deployments" }
 
 func (c *JobDeploymentsCommand) Run(args []string) int {
-	var json, latest, verbose bool
+	var json, latest, verbose, all bool
 	var tmpl string
 
 	flags := c.Meta.FlagSet(c.Name(), FlagSetClient)
 	flags.Usage = func() { c.Ui.Output(c.Help()) }
 	flags.BoolVar(&latest, "latest", false, "")
 	flags.BoolVar(&verbose, "verbose", false, "")
+	flags.BoolVar(&all, "all", false, "")
```

Comment:
Don't forget to update Autocomplete flags and help text/description above.

Reply:
done